Templates/HttpPUT(CRUD)-FSharp/run.fsx

```diff
@@ -1,23 +1,39 @@
 ﻿#r "System.Net.Http"
 #r "Microsoft.WindowsAzure.Storage"
+#r "Newtonsoft.Json"
 
 open System
 open System.Net
 open System.Net.Http
 open Microsoft.WindowsAzure.Storage.Table
+open Newtonsoft.Json
+open Newtonsoft.Json.Linq
 
-type Person() =
-    inherit TableEntity()
-    member val Name: string = null with get, set
+type Person(name, partitionKey, rowKey) =
+    inherit TableEntity(partitionKey, rowKey)
+    member val Name: string = name with get, set
 
-let Run(person: Person, outTable: CloudTable, log: TraceWriter) =
-    if String.IsNullOrEmpty(person.Name) then
-        let response = new HttpResponseMessage(HttpStatusCode.BadRequest)
-        response.Content <-
-            new StringContent("A non-empty Name must be specified.")
-        response
-    else
-        log.Info(sprintf "PersonName=%s" person.Name)
-        let updateOperation = TableOperation.InsertOrReplace(person)
-        let result = outTable.Execute(updateOperation)
-        new HttpResponseMessage(enum<HttpStatusCode>(result.HttpStatusCode))
+let Run(req: HttpRequestMessage, outTable: CloudTable, log: TraceWriter) =
```

Comment:
Why does POCO deserialization no longer work? It should - do we have a bug? Have you tried on the latest bits?